frontend/src/components/samples/SamplesPage.js

```diff
@@ -3,7 +3,7 @@ import React from "react";
 import {Redirect, Route, Switch} from "react-router-dom";
 
 import SampleEditContent from "./SampleEditContent";
-import SamplesDetailContent from "./SamplesDetailContent";
+import SamplesDetailContent from "./sampleDetails/SampleDetailsContent";
```

Comment:
Keep import name and filename identical for consistency.

Reply:
Also new directory could be just `details` to avoid too much `sample` repetition.